index.html

```diff
@@ -1,13 +0,0 @@
-<!DOCTYPE html>
```

Comment:
- [ ]  Let's make sure we make our Pull request from `feature_branch` to `main`, in our case, we reversed it, Let's please fix that by making a new pull request with same everything but add in PR description link to this pull request explaining to your next code reviewer what happened :shipit: